.prettierrc.js

```diff
@@ -0,0 +1,5 @@
+'use strict';
+
+module.exports = {
+  singleQuote: true,
```

Comment:
nit: add `trailing-comma`?

Reply:
Since this technically isn't our code (it's forked) I wanted to leave it as vanilla as possible.

If the original is pretty much abandonware, and we're not going to submit any of this upstream or anything, then we should consider this and other changes we'd like to make.

It raises the bigger question though—what impact does the state of this addon (and the other that enable it) have on how we deploy? Do we want to keep patching this, or look for alternatives, something else? (it's working fine so no need to do anything really, but food for thought).